### ISSUE TITLE
Suggestion: Alternative runway filters

### ISSUE THREAD

**Issue by daheise** [kaosfere/nftools#1]
Would you be open to suggestions and/or PRs for different runway filters? I think your current filter for lights leaves only ~7k airports worldwide. That may be chopping out more airports than desirable. Two I've thought of are like the following:

Delete short grass runways (length here is takeoff length for a King Air 350):
`SELECT count(*) from AIRPORT where num_runway_soft > 0 and longest_runway_length < 3300;`
Result: 21,420 airports remain

Delete airports without at least one hard runway:
SELECT count(*) from AIRPORT where num_runway_hard == 0
Result: 16,602 air ports remain

**Comment by kaosfere** [comment]
Yeah...   the current approach is a pretty blunt hammer.   I'm torn on adding more complex filtering...  I like it and might even use it myself, but at some point doing all that on the command line is going to get pretty unwieldy.  I'm not sure how best to manage that.

I'd like to have a simple UI around this at some point which would help, but even prior to then a not TOO complex way of performing more complicated filtering could be useful.   Hmm.
